code/modules/projectiles/gun.dm

```diff
@@ -796,6 +791,8 @@
 	pickup_sound = null
 	equip_sound = null
 
+	var/static/list/equippable_slots = list(slot_l_hand, slot_r_hand) // putting it here instead of in the proc so we don't have to init a shitton of lists
```

Comment:
This doesn't really matter as objects created in a proc are cleaned up at the end. Also, what you described in that comment is what `static` does, and it works in a proc body.